WordPress/src/main/res/layout/domain_suggestions_fragment.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                xmlns:app="http://schemas.android.com/apk/res-auto"
+                xmlns:tools="http://schemas.android.com/tools"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="@android:color/white">
+
+    <LinearLayout
+        android:id="@+id/search_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginTop="@dimen/margin_extra_small"
+        android:gravity="center_vertical"
+        android:orientation="horizontal"
+        android:paddingBottom="@dimen/margin_extra_large"
+        android:paddingEnd="@dimen/margin_extra_large"
+        android:paddingStart="@dimen/margin_large"
+        android:paddingTop="@dimen/margin_extra_large">
+
+        <FrameLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/margin_extra_extra_medium_large">
+
+            <ImageView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/margin_extra_extra_small"
+                android:contentDescription="@null"
+                android:src="@drawable/ic_search_white_24dp"
+                android:tint="@color/neutral"/>
+
+            <ProgressBar
+                android:id="@+id/suggestion_progress_bar"
+                android:layout_width="@dimen/progress_spinner_small"
+                android:layout_height="@dimen/progress_spinner_small"
+                android:background="@android:color/white"
+                tools:visibility="gone"/>
+        </FrameLayout>
+
+        <EditText
+            android:id="@+id/domain_suggestion_keyword_input"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/margin_extra_small"
+            android:background="@android:color/transparent"
+            android:clickable="false"
+            android:hint="@string/suggestions_search_hint"
+            android:imeOptions="actionDone"
+            android:inputType="text"
+            android:maxLines="1"
+            android:paddingStart="@dimen/margin_extra_extra_small"
+            android:textColorHint="@color/neutral_400"
+            android:textSize="@dimen/text_sz_medium"
+            tools:ignore="Autofill,RtlSymmetry"/>
+    </LinearLayout>
+
+    <View
+        android:id="@+id/divider"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
```

Comment:
```suggestion
        android:layout_height="@dimen/list_divider_height"
```

We may want to revert `1dp` back to `@dimen/list_divider_height` to ensure all dividers behave the same.  Also, the `list_divider_height` value is `1dp` so it's effectively the same at the moment.